src/app/components/navigation/navigation.component.ts

```diff
@@ -17,7 +17,7 @@ export class NavigationComponent implements OnInit {
 
   constructor(private translateService: TranslateService, zone: NgZone) {
     this.translateService.setDefaultLang(this.lang_default);
-    this.lang = this.translateService.getBrowserLang();
+    this.lang = localStorage.getItem('lang') || this.translateService.getBrowserLang();
```

Comment:
Where is this local storage? Is the local storage bound to where the URL where the GUI was loaded from?

Reply:
It is bound the origin of the web page so essentially the protocol, domain and port used to load it.

https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage

Reply:
And that is a standard so that it works on any browser, or at least does not create problem on other browsers?

Reply:
It is a standard, and as you could see on the page linked above, it is very well supported.

https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage#browser_compatibility